pkg/cmd/cli/cmd/newapp.go

```diff
@@ -169,8 +171,12 @@ func RunNewApplication(f *clientcmd.Factory, out io.Writer, c *cobra.Command, ar
 	for _, item := range result.List.Items {
 		switch t := item.(type) {
 		case *kapi.Service:
-			// TODO: handle multi-port created services
-			fmt.Fprintf(c.Out(), "Service %q created at %s:%d to talk to pods over port %d.\n", t.Name, t.Spec.PortalIP, t.Spec.Ports[0].Port, t.Spec.Ports[0].TargetPort.IntVal)
+			var portals, targetPorts []string
+			for _, port := range t.Spec.Ports {
+				portals = append(portals, fmt.Sprintf("%s:%d", t.Spec.PortalIP, port.Port))
```

Comment:
I'd prefer to show port mappings if they exist.  See describeServicePorts in projectstatus.go for the preferred output form.